cuda_bindings/docs/source/install.md

```diff
@@ -44,13 +44,17 @@ $ conda install -c conda-forge cuda-python
 ### Requirements
 
 * CUDA Toolkit headers[^1]
+* static CUDA runtime[^2]
```

Comment:
cc @isvoid for vis

Reply:
Does it make more sense to call this the `CUDA Runtime Static Library`? Also `Runtime` is capitalized in NVIDIA docs, so capitalizing here.
```suggestion
* CUDA Runtime static library[^2]
```